packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2708,6 +2690,36 @@ export interface MediaDiagnosticsState {
     latest: LatestMediaDiagnostics;
 }
 
+// @beta
+export interface Mention {
+    displayText: string;
+    icon?: JSX.Element;
+    id: string;
+}
+
+// @beta
+export interface MentionDisplayOptions {
+    onRenderMention?: (mention: Mention) => JSX.Element;
+}
+
+// @beta
+export interface MentionLookupOptions {
+    onQueryUpdated: (query: string) => Promise<Mention[]>;
+    onRenderSuggestionItem?: (suggestion: Mention, onSuggestionSelected: (suggestion: Mention) => void) => JSX.Element;
+    trigger?: string;
+}
+
+// @beta
+export type MentionOptions = {
+    lookupOptions?: MentionLookupOptions;
+    displayOptions?: MentionDisplayOptions;
+};
+
+// @beta
+export interface MentionPopoverStrings {
+    mentionPopoverHeader: string;
```

Comment:
It sounds pretty similar to https://developer.microsoft.com/en-us/fluentui#/controls/web/callout from fluent? Is it the same thing or different one? If it is the same, let's align the name for that